pkgs/applications/misc/veracrypt/default.nix

```diff
@@ -4,12 +4,12 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   pname = "veracrypt";
-  version = "1.23";
-  minorVersion = "-Hotfix-2";
+  version = "1.24";
+  hotfix = "-Hotfix1";
 
   src = fetchurl {
-    url = "https://launchpad.net/${pname}/trunk/${version}/+download/VeraCrypt_${version}${minorVersion}_Source.zip";
-    sha256 = "229de81b2478cfa5fa73e74e60798a298cd616e9852b9f47b484c80bc2a2c259";
+    url = "https://launchpad.net/${pname}/trunk/${version}${toLower hotfix}/+download/VeraCrypt_${version}${hotfix}_Source.tar.bz2";
```

Comment:
Then:
```suggestion
    url = "https://launchpad.net/${pname}/trunk/${toLower version}/+download/VeraCrypt_${version}_Source.tar.bz2";
```

Reply:
Addressed in cffaa114cac2a2d0e448d8ef81041b4f644903cf